src/components/MdxComponents/Contributors.tsx

```diff
@@ -1,14 +1,34 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { useStaticQuery, graphql } from 'gatsby';
 import { css } from '@emotion/core';
 import { useTheme } from 'emotion-theming';
+import { Remark } from 'react-remark';
+import ContributorsDetail from './ContributorsDetail';
+import styled from '@emotion/styled';
+
+const ThankyouWrapper = styled.div`
+  display: block;
+  margin: 40px 0;
+  width: 100%;
+`;
+
+const NoteWrapper = styled.div`
+  display: block;
+  margin: 60px 0 50px 0;
+  width: 100%;
+`;
 
 const style = theme => css`
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
+  .wrapper {
+    display: flex;
+    flex-direction: row;
```

Comment:
@mildronize just FYI flex-direction is row by default, later you don't need to specify this value.

Reply:
The existing code use `flex-direction: row;` , No need to edit, I just refactor it.